contributing/development/compiling/compiling_for_ios.rst

```diff
@@ -16,6 +16,9 @@ Requirements
 - `Python 3.6+ <https://www.python.org/downloads/macos/>`_.
 - `SCons 3.1.2+ <https://scons.org/pages/download.html>`_ build system.
 - `Xcode <https://apps.apple.com/us/app/xcode/id497799835>`_.
+    - Launch Xcode once and install iOS support.
```

Comment:
It might be worth mentioning that you can also do it in the `Settings..` → `Platforms` if you already installed Xcode and skipped iOS support during first launch.